src/qibo/gates/abstract.py

```diff
@@ -366,6 +366,9 @@ def matrix(self, backend=None):
 
         return backend.matrix(self)
 
+    def clifford_operation(self, backend=CliffordBackend()):
```

Comment:
does `backend` as input here make sense? Clifford backend seems to be the only option